lib/libzrepl/mgmt_conn.c

```diff
@@ -1017,6 +1159,32 @@ process_message(uzfs_mgmt_conn_t *conn)
 		    payload_size);
 		break;
 
+	case ZVOL_OPCODE_SNAP_LIST:
+		if (payload_size == 0 || payload_size > MAX_NAME_LEN) {
```

Comment:
this code seems to be repetition of SNAPCREATE opcode. Need to remove duplicate code.